src/catalog/src/builtin.rs

```diff
@@ -8142,70 +8143,87 @@ pub static MZ_CONSOLE_CLUSTER_UTILIZATION_OVERVIEW: LazyLock<BuiltinView> = Lazy
     cluster_id
   FROM mz_catalog.mz_cluster_replicas
 ),
+replica_metrics_history AS (
+  SELECT
+    m.occurred_at,
+    m.replica_id,
+    r.size,
+    (SUM(m.cpu_nano_cores::float8) / (s.cpu_nano_cores * s.processes)) AS cpu_percent,
```

Comment:
It might be nice to pull out the total resource calculation, e.g. `s.cpu_nano_cores * s.processes` into its own CTE, but up to you!